local-dev/skaffold.yaml.template

```diff
@@ -0,0 +1,22 @@
+#This is a Skaffold configuration, which lets developers continuously push new images to their development namespaces.
+apiVersion: skaffold/v2alpha4
+kind: Config
+build:
+  artifacts:
+  - image: gcr.io/terra-kernel-k8s/terra-crl-janitor
+    context: ../
+    jib: {}
+deploy:
+  helm:
+    releases:
+      - name: crljanitor-ENV
+        namespace: terra-ENV
```

Comment:
Can we use the helm values instead here and then only sed replace the values.yaml.template (and make this `skaffold.yaml` instead of a template?
`-name: crljanitor-{{ .Values.global.terraEnv }}`

Reply:
Nope, Skaffold doesn't have that context

Reply:
Kind of unfortunate. My 1 min uninformed read is there's some integration? https://skaffold.dev/docs/pipeline-stages/deployers/helm/
Hopefully it doesn't matter much for local configuration, but it's yet another configuration template variation with ad-hoc sed tooling.